README.md

```diff
@@ -153,9 +153,34 @@ let balance = try! xpringClient.getBalance(for: address)
 print(balance) // Logs a balance in drops of XRP
 ```
 
+### Checking Transaction Status
+
+A `XpringClient` can check the status of an operation on the XRP Ledger. 
+
+Transaction states can be:
```

Comment:
I know this is fraught, but maybe we should rephrase this to indicate that these states are _what XpringKit returns_.  I'd be inclined to add a link or two, too, like so (also tweaking the descriptions a bit):

> XpringKit returns the following transaction states:
> - `succeeded`: The transaction was successfully validated and applied to the XRP Ledger.
> - `failed:` The transaction was successfully validated but not applied to the XRP Ledger. Or the operation will never be validated.
>  - `pending`: The transaction has not yet been validated, but may be validated in the future.
>  - `unknown`: The transaction status could not be determined.
> 
> **Note:** For more information, see [Reliable Transaction Submission](https://xrpl.org/reliable-transaction-submission.html) and [Transaction Results](https://xrpl.org/transaction-results.html).


I also changed "operation" to "transaction" throughout to be consistent and because I think the operations are transactions in this case, but feel free to change if that's not correct.